includes/Form.class.php

```diff
@@ -165,6 +165,18 @@ function process($params) {
 		}
 		$email = $_POST["email"];
 		$phone = isset($_POST["phone"]) ? $_POST["phone"] : "";
+		$lists = (isset($_POST["nlbox"]) && $_POST["nlbox"]) ? $_POST["nlbox"] : array();
+		if (isset($_SESSION["image_random_value"])) {

```

Comment:
oh i'm talking about a hacker. imagine i (not client, or their user, but a random dude) do this:
<?php require('http.php'); ac_http_post($theirprocessurl, array('my post stuff here'); ?>

when i run my script from my computer, it will do a call to $theirprocessurl (the script that runs your process() method). My script is hitting it for the first time, it is not sending over any cookie info, so the server will not pick up any existing sessions, but generate a blank new session for my request. it will do that every single time i run my script -- give me a new session. that session wont have that var setup, so the check will never kick in.